samples/embedded-sign-in-widget/src/main/java/com/okta/spring/example/controllers/HomeController.java

```diff
@@ -40,6 +44,10 @@ public String home(
                     String.format("/oauth2/authorization/okta?error=%s&state=%s", error, state);
             return "redirect:" + oauthAuthUri;
         }
+        //cleanup the context in case of unsuccessful login
+        if (Strings.hasText(error)) {
```

Comment:
This will also clear the context if we get `error=interaction required` which is not an actual error but an indication that we need to complete more steps to get interaction code.